test/data/css/customProperties.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <style>
+      .test-custom-props {
+          --prop1: val1;
+          --prop2:val2;
+          --prop3:val3 ;
+          --prop4: "val4" ;
+      }
+    </style>
+  </head>
+  <body>
+    <div class="test-custom-props"></div>
+    <script src="../../jquery.js"></script>
+    <script src="../iframeTest.js"></script>
+    <script>
+      startIframeTest( jQuery( ".test-custom-props" ) );
```

Comment:
> Couldn't it be add to our test css?

In here - https://github.com/jquery/jquery/blob/02c5e2908c2d99f191f5ec9e365ef2a3d357d1eb/test/data/testsuite.css i mean